src/installation/guides/chroot.md

```diff
@@ -0,0 +1,222 @@
+# Installation via chroot (x86 or x86_64)
+
+## Prepare Filesystems
+
+These instructions can be followed on any running Linux system that has xbps
+installed, including an already working installation of Void or an official
+[live image](../live-images/prep.md). It is assumed that the host computer is
+connected to the Internet.
+
+[Partition your disks](../live-images/partitions.md) and format them using
+[mke2fs(8)](https://man.voidlinux.org/mke2fs.8),
+[mkfs.xfs(8)](https://man.voidlinux.org/mkfs.xfs.8),
+[mkfs.btrfs(8)](https://man.voidlinux.org/mkfs.btrfs.8) or whatever tools are
+necessary for your filesystem(s) of choice.
+
+[cfdisk(8)](https://man.voidlinux.org/cfdisk.8) and
+[fdisk(8)](https://man.voidlinux.org/fdisk.8) are available on live
+distributions for partitioning, but you may wish to use
+[gdisk(8)](https://man.voidlinux.org/gdisk.8) (from the package `gptfdisk`) or
+[parted(8)](https://man.voidlinux.org/parted.8) instead.
+
+> Note: For a UEFI booting system, make sure to create a UEFI System Partition
+> (ESP). The ESP should have the partition type "EFI System" (code `ef00`) and
+> be formatted as fat32 using
+> [mkfs.vfat(8)](https://man.voidlinux.org/mkfs.vfat.8).
+
+### Create a New Root and Mount Filesystems
+
+This guide will assume the new root filesystem is mounted on `/mnt`. You may
+wish to mount it elsewhere.
+
+If using UEFI, mount the EFI System Partition as `/mnt/boot/efi`.
+
+For example, if `/dev/sda2` is to be mounted as `/` and `/dev/sda3` is to be
+booted as `/home` and `dev/sda1` is the EFI System Partition
```

Comment:
Perhaps specify that a separated /home partition is optional? I don't know exactly what level of knowledge we are aiming at here, but it could confuse people.

Reply:
I did that originally so that it was clear how to deal with having more than one partition. The example setup was for BIOS boot, but then I later changed it to UEFI which means you need a separate partition for ESP anyway, so /home isn't doing anything to help anymore.

There was also a longer section on partitioning that I gutted because it's out of scope of the guide that ended with recommending a single partition if the reader was unsure. That last part is probably still useful in this guide. It would make sense to use this recommended partitioning scheme as the example throughout the rest of the guide.

Reply:
That's fair. We could just say that a separate home partition isn't necessary, it's used for making life easier if reinstalling, and be done with it.

In regards to this section, your language is a bit inconsistent. "mounted as" and "booted as", I think you should stick to "mounted as". Perhaps make a table for it?